services/activitylog/pkg/service/l10n/.tx/config

```diff
@@ -4,6 +4,7 @@ host = https://www.transifex.com
 [o:owncloud-org:p:owncloud:r:ocis-activitylog]
 file_filter = locale/<lang>/LC_MESSAGES/activitylog.po
 minimum_perc = 75
+resource_name = ocis-activitylog
```

Comment:
this change has nothing to do with the pr title ...

update docs !== change transifex configuration

Reply:
fixed